libraries/AP_GPS/AP_GPS.h

```diff
@@ -135,9 +137,12 @@ class AP_GPS
         uint32_t time_week_ms;              ///< GPS time (milliseconds from start of GPS week)
         uint16_t time_week;                 ///< GPS week number
         Location location;                  ///< last fix location
+        uint8_t atteulerMode;               ///< mode in which the Attitude is computed
         float ground_speed;                 ///< ground speed in m/sec
         float ground_course;                ///< ground course in degrees
         float gps_yaw;                      ///< GPS derived yaw information, if available (degrees)
+        float gps_pitch;                    ///< GPS derived pitch information, if available (degrees)
```

Comment:
First of all thank you for reviewing this so quickly.

Yes you right, i tough maybe for future use because integrating the roll and pitch into the kalman-filter is something out of my skill. I will delete them but knowing this is supported by the AsterRX-i S.

Reply:
it might be worth having them if we log them. As GPS modules that can give attitude become more available the first step would be to log the values and compare to what we get with our existing EKF code.
Note sure what log message - maybe a new log msg GPI, for "GPS IMU" ?